README.rst

```diff
@@ -76,25 +76,33 @@ examples in the examples directory.
 Getting the package
 ===================
 
-General Build and Installation instructions for ETS are available here:
+The mayavi codebase can be found in github:
 
- http://github.enthought.com/mayavi/mayavi/installation.html
+https://github.com/enthought/mayavi
 
-Source tarballs for all stable ETS packages are available at
+General Build and Installation instructions are available `here 
+<http://docs.enthought.com/mayavi/mayavi/installation.html#installing-ready-made-distributions>`_
 
- http://code.enthought.com/enstaller/eggs/source
+Source tarballs for necessary stable ETS packages are available through pypi
 
+- `traits <https://pypi.python.org/pypi/traits>`_
+- `traitsui <https://pypi.python.org/pypi/traitsui>`_
+- `pyface <https://pypi.python.org/pypi/pyface>`_
+- `apptools <https://pypi.python.org/pypi/apptools>`_
+- `envisage <https://pypi.python.org/pypi/envisage>`_
+- `mayavi <https://pypi.python.org/pypi/mayavi>`_
 
-Documentation
-==============
+Development versions exist in the github `Enthought organization <https://github.com/enthought>`_
 
-More documentation is available in the online user manual, 
 
-http://github.enthought.com/mayavi/mayavi
+Documentation
+==============
 
-or in `docs` directory of the sources.  This includes a man page for the
-`mayavi2` application, a users guide in HTML and PDF format and
-documentation for `mlab`.
+More documentation is available in the `online user manual 
+<http://docs.enthought.com/mayavi/mayavi/>_` or in `docs` directory 
```

Comment:
This link `<http://docs.enthought.com/mayavi/mayavi/>_` causes a `404` due to the extraneous  `>`.